### ISSUE TITLE
Topic poll projection not functioning

### ISSUE THREAD

**Issue by luisa-beerboom** [OpenSlides/openslides-client#1186]
Topic poll was implemented in #1010, however the projection-feature is still not implemented/broken and shows an empty slide while throwing an error "slide error:  calculating key [...]" on the slide-container.component.ts

Might not be a pure frontend problem.

**Comment by emanuelschuetze** [comment]
error message from autoupdate service:
`docker-autoupdate-1        | 2022/05/25 14:01:04 Error calculating key projection/52/content: calculating slide poll for p7on &{52  poll/8 1 [123 125]}: get Options func: getTitleInfoFromContentObject: no titler function registered for`

**Comment by jsangmeister** [comment]
@ostcar I suspect that the autoupdate service/restricter ist not ready yet for topic polls, can you look into this?

**Comment by jsangmeister** [comment]
I think we should make the content object required, then. Just accessing the content object from the poll if the one from the option is empty is probably too specific to be a valid solution, right?

**Comment by jsangmeister** [comment]
Update: I just looked into it again, and I didn't fully grasp the problem. For options, it is okay if *either* `content_object_id` *or* `text` is set, which was done because of things such as topic polls. There, each options contains a free-hand text to vote about instead of some content object. Therefore, it would not make sense to fill the field `content_object_id`.

Instead, I would adjust the wiki at https://github.com/OpenSlides/OpenSlides/wiki/Projector-System#poll. It also says to leave out certain keys sometimes, so we should leave out `text` if `option/text` is empty and leave out `content_object` if `option/content_object_id` is empty (so the `GetTitleInformation` function is not called at all, but also not needed, since we have the text to display).

After that, a specific slide has to be created for topic polls and inserted into https://github.com/OpenSlides/openslides-client/blob/main/client/src/app/site/pages/meetings/modules/projector/modules/slides/components/poll-slide/components/poll-slide.component.html. This can just display the given key `text` for the options.

Let me know if I overlooked something, else you can implement this and I will adjust the wiki accordingly.

**Comment by GabrielInTheWorld** [comment]
> Update: I just looked into it again, and I didn't fully grasp the problem. For options, it is okay if _either_ `content_object_id` _or_ `text` is set, which was done because of things such as topic polls. There, each options contains a free-hand text to vote about instead of some content object. Therefore, it would not make sense to fill the field `content_object_id`.
> 

Yes, that's right. The topic polls were implemented correctly. The autoupdate service has to be updated accordingly, to deal with topic polls (respectively an empty `content_object_id` but `text`).

**Comment by ostcar** [comment]
I don't have any opinion about this. Just tell me as abstract as possible how the autoupdate-service should render the content in which situation.

**Comment by jsangmeister** [comment]
I adjusted https://github.com/OpenSlides/OpenSlides/wiki/Projector-System#poll as described above - should be a simple fix.

**Comment by GabrielInTheWorld** [comment]
@ostcar or @r-peschke can you do this in the projector?
